res/css/components/views/location/_ShareType.scss

```diff
@@ -24,6 +24,36 @@ limitations under the License.
     padding: 60px $spacing-12 $spacing-32;
 
     color: $primary-content;
+
+    .mx_ShareType_wrapper_options {
+        display: flex;
+        flex-flow: column;
```

Comment:
`flex-flow` is a shorthand for `flex-direction` and `flex-wrap`.
Since you are using only a single prop here, it should better be `flex-direction`.

Reply:
Right, that makes sense as well.